### ISSUE TITLE
FMLAT not loading from dependencies

### ISSUE THREAD

**Issue by Ivorforce** [MinecraftForge/MinecraftForge#1437]
So, we have a project in which there is a dependency in the build.gradle closure like this:

dependencies {
   [...]
    compile 'ivorius.reccomplex:RecurrentComplex:0.9.2.1:dev'
}

Now, RecurrentComplex:0.9.2.1 contains an FMLAT whereas the build.gradle looks like [this](https://github.com/Ivorforce/RecurrentComplex/blob/master/build.gradle).

The downloaded jar contains a Manifest in which the FMLAT correctly points to the access transformer ('FMLAT: recurrentcomplex_at.cfg').
Likewise, the AT itself is contained the META-INF folder of the jar.

Long story short: The FMLAT does not load. It loads when put in the mods folder, and from the original dev environment however.
